en/hosting_guide/server_provisionning.rst

```diff
@@ -0,0 +1,77 @@
+Server provisionning
+====================
+
+Context
+-------
+
+Believe it or not, at some time, your Open Orchestra project will require an external
+server to be deployed on.
```

Comment:
This tone doesn't bother me, but it's true that is contrasts with the rest of the doc